index.md

```diff
@@ -72,6 +72,17 @@ can either be static or implement synaptic plasticity rules such as
 * [Synapses in NEST: An overview](synapses_overview)
 * [Developing synapse models](synapse_models)
 
+## Updating models from 2.14 or prior to 2.16
+
+With the introduction of the new connection infrastructure of the [5g kernel](https://www.frontiersin.org/articles/10.3389/fninf.2018.00002/full), 
+neuron, synapse and device models need to be slightly adapted from prior 
```

Comment:
@jakobj @suku248  Should we make this "rate neuron, synapse, and device models"?

Reply:
Done.